docker-entrypoint.sh

```diff
@@ -17,6 +17,18 @@ if [ "$DOCKER_CONFIG" ]; then
     unset DOCKER_CONFIG
 fi
 
+if [ "$GCP_PROJECT" ]; then
+    mkdir -p ~/.config/gcloud/configurations/
+    cat <<- 'HERE' > ~/.config/gcloud/configurations/config_default
```

Comment:
Did somebody say time for HEREDOC? @mattrobenolt please check if these changes address your concerns :)

Reply:
Hell yeah, if this works, that's great. If not, we can go to passing in the entire configuration file through as well if we need to.

Reply:
this needs to be ~freight instead of ~, that'll fix the problem we're talking about in #ops

Reply:
.... or for super fancy style points you can create the file for one user and copy it to the other so both can do it

Reply:
I'd suggest doing it like: https://github.com/getsentry/freight/blob/master/docker-entrypoint.sh#L13-L15

That way permissions are also correct, etc.